README.md

```diff
@@ -3,47 +3,52 @@
 [EIP2771-compatible](https://eips.ethereum.org/EIPS/eip-2771) transaction relayer library for Ethereum. The core of the library is bindings to a compatible forwarder contract. 
 
 Components:
-- `cmd`: example CLI that uses examples/minimal_forwarder
-- `examples`: example implementations of forwarder and recipient contracts, as well as Go bindings, required interface implementations, and unit tests
-	- `examples/gsn_forwarder`: an implementation using OpenGSN's [forwarder contract](https://github.com/opengsn/gsn/tree/master/packages/contracts/src/forwarder).
-	- `examples/minimal_forwarder`: an implementation using OpenZeppelin's [minimal forwarder](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/metatx/MinimalForwarder.sol).
-	- `examples/mock_recipient`: an implementation of a EIP2771-compatible recipient contract which can receive calls from a forwarder.
+- `cmd/relayer`: example CLI that uses impls/minimal_forwarder
+- `impls`: example implementations of forwarder and recipient contracts, as well as Go
+  bindings, required interface implementations, and unit tests
+	- `impls/gsn_forwarder`: an implementation using OpenGSN's [forwarder contract](https://github.com/opengsn/gsn/tree/master/packages/contracts/src/forwarder).
+	- `impls/minimal_forwarder`: an implementation using OpenZeppelin's [minimal forwarder](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/metatx/MinimalForwarder.sol).
+- `examples/mock_recipient`: an implementation of a EIP2771-compatible recipient contract
+  which can receive calls from a forwarder.
 - `relayer`: functionality to call forwarder contracts
 - `rpc`: rpc server for an end-user to submit txs to, accepts a `*relayer.Relayer`
 
 ## Requirements
 
 - go 1.19+
-- abigen: can install with `bash scripts/install-abigen.sh`
+- abigen: can install using `./scripts/install-abigen.sh`
 
 ## Usage
 
 ### As an application
 
-See `cmd/main.go` for an example app using the `examples/minimal_forwarder` package. 
+See `cmd/relayer/main.go` for an example app using the `impls/minimal_forwarder` package. 
```

Comment:
I was just searching and updating the word `examples`, but I think this needs other updates.  Should that say `gsnforwarder`?

Reply:
yeah i changed the default app to use gsnforwarder, so yes